scss/_custom.scss

```diff
@@ -970,10 +970,11 @@ $breadcrumb-padding-y:          .75rem !default;
 $breadcrumb-padding-x:          1rem !default;
 $breadcrumb-item-padding:       .5rem !default;
 
-$breadcrumb-bg:                 $gray-lighter !default;
+$breadcrumb-bg:                 transparent !default;
```

Comment:
Only variables using the default configuration value should use the "!default" mark